modules/gitops-installing-argo-rollouts-cli-by-using-linux.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-cli-by-using-linux_{context}"]
+= Installing Argo Rollouts CLI by using Linux
+
+If you are a Linux user, you can install Argo Rollouts CLI by performing the steps described in this procedure.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (oc).
+
+.Procedure
+
+. Download the latest version of the Argo Rollouts CLI binary, `kubectl-argo-rollouts`, by running the following command:
++
+[source,terminal]
+----
+$ curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+----
+
+. Ensure that the `kubectl-argo-rollouts` binary is executable by running the following command:
++
+[source,terminal]
+----
+$ chmod +x ./kubectl-argo-rollouts-linux-amd64
+----
+
+. Move the `kubectl-argo-rollouts` binary to the system path by running the following command.
++
+[source,terminal]
+----
+$ sudo mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
+----
+
+[NOTE]
+====
+Ensure that you have `superuser` privileges to run this command.
+====
+
+.Verification
+
+. Verify that the plugin is installed correctly by running the following command:
++
+[source,terminal]
+----
+$ oc argo rollouts version
+----
++
+.Example output
+[source,terminal]
+----
+kubectl-argo-rollouts: v1.6.6+737ca89
+  BuildDate: 2024-02-13T15:39:31Z #<1>
+  GitCommit: 737ca89b42e4791e96e05b438c2b8540737a2a1a
+  GitTreeState: clean
+  GoVersion: go1.20.14 #<2>
+  Compiler: gc
+  Platform: linux/amd64 #<3>
+----
+<1> - Specifies the build date information of the Argo Rollouts binary
+<2> - Specifies the version of the `Go` language used for building the Argo Rollouts binary
+<3> - Specifies the platform used for building the Argo Rollouts binary
```

Comment:
When I mentioned to add the callouts, I was mostly thinking to add callouts that would help user to determine what to look for here, because it is a verification step. The added callouts are good, I just want it to be clear how do we determine from this that the installation is correct.

Is just the fact that the output is there (the version command worked) enough of the verification? That is what I was trying to ask. Or if the user should look for the specific relevant information from the output that would help determine if the installation is correct.

Reply:
If just the fact that the command worked is the verification step by itself, then I would just tweak the step sentence a little bit:
. Verify that the plugin is installed correctly by running the following command and receiving the similar output:

Reply:
Yeah, the idea is that this output is displayed after running this command is proof that the installation is successful. I will change the sentence slightly as per your suggestion @eromanova97. Thanks :)